Library/Formula/napi.rb

```diff
@@ -0,0 +1,35 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter"
+  homepage "https://github.com/dagon666/napi"
+  url "https://github.com/dagon666/napi/archive/v1.3.4.tar.gz"
+  sha256 "32b8ee39e00e9ed8cbbdb02bb683ccbfb02bd1f109b1a1e9f97e5badbf12c996"
+
+  depends_on "ffmpeg" => :required
```

Comment:
This should just be `depends_on "ffmpeg"`. Should also solve your `pkg-config` failure message.